Developer_Documentation/1.6/code_docs/modules/order_total_modules.md

```diff
@@ -0,0 +1,334 @@
+# Introduction
+Zen Cart *order_total* modules are used to calculate (and display) sub-totals, taxes, totals and other intermediary values for an order. *Order-total* modules can also calculate discounts or other subtractions from the amount a customer owes for an order. An *order-total* includes, at a minimum, two files:
+
```

Comment:
Actually, I see dozens of places were you use "order-total" as though you mean "the module" itself. That's got me to thinking about how best to convey the meaning.  Given that the words "order" and "total" used together could get confused with "the total of an order", or "the output of said order-total module calculation", or as you've used it: "the module/class itself", I wonder if simply using "module" (instead of "order-total" or "order-total module") when referring to the module, may bring the most clarity to the reader?